scripts/train.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# 
+# This script does the following:
+# 1. Creates a virtual environment and installs the required dependencies
```

Comment:
Maybe you can specify that the env must be named `venv_monai`

Reply:
Done! https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/90/commits/d37f25524701052bab7fe94739b5e1fa63e908ef